src/adapters/bam/llm.ts

```diff
@@ -63,6 +65,8 @@ export interface BAMLLMOutputConstructor {
   moderations?: BAMLLMOutputModeration | BAMLLMOutputModeration[];
 }
 
+const MAX_EMBEDDING_INPUTS = 20;
```

Comment:
Inline or add to custom embedding options.

Reply:
moved inline: https://github.com/i-am-bee/bee-agent-framework/pull/210/commits/bbcca1ecbe18a2271c6d70a831e31e3cdd42f034